sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -594,17 +594,21 @@ private static void printVerdict(final long numberOfTestFailures, final File tes
         }
 
         if (numberOfTestFailures == 0) {
-            LOG.info(
-                    "Test run with {} Tests was completed successfully. No problems were found.",
-                    testRunObserver.getTotalNumberOfTestsRun());
+            if (testRunObserver.isInvalid()) {
+                LOG.info(
+                        "Test run with {} Tests was completed successfully but other problems were found. Please consult the logfiles in {} for further information.",
```

Comment:
I wouldn't call an invalid test run a **successfully** completed test run. Definitely it is not successful when it is invalid.

Reply:
i agree - done